packages/test-suite/src/values.js

```diff
@@ -56,6 +56,24 @@ const keyvValueTests = (test, Keyv, store) => {
 		t.deepEqual(await keyv.get('foo'), value);
 	});
 
+	test.serial('value can not be symbol', async t => {
+		const keyv = new Keyv({ store: store() });
+		const value = Symbol('value');
+		try {
+			await keyv.set('foo', value);
+		} catch (error) {
+			t.is(error.context, 'symbol cannot be serialized');
+		}
+	});
+
+	test.serial('value can be bigint', async t => {
+		const keyv = new Keyv({ store: store() });
+		// eslint-disable-next-line no-loss-of-precision
+		const value = 9_223_372_036_854_775_807;
```

Comment:
This is not a bigint; this is just a normal JS number that's so large that it gets immediately rounded.

You can replace this line with
```
const value = BigInt('9223372036854775807');
```

and then the test will fail because of the issue I describe in my other comment.

If you want to be able to round-trip BigInts through JSON, you need an unambiguous way to represent them in the protocol, or a way of deciding when numbers should be BigInts.

My suggestion is that this change (which mutates the global BigInt, which is icky too) be rolled back, and that folks who want to use bigints supply a custom serializer (perhaps using https://www.npmjs.com/package/json-bigint or something).

Reply:
@glasser there is already an option to supply a custom serializer.

Reply:
I understand; my point is that since "JSON encoding of bigints" is not particularly standardized, using a custom serializer might be more appropriate for handling them. (Alternatively, the default serializer could perhaps use something other than JSON that natively supports more types.)